src/test/java/com/teragrep/pth10/BloomFilterOperationsTest.java

```diff
@@ -132,4 +140,148 @@ public void estimateTest() {
                         }
                 );
     }
+
+    @Test
+    @DisabledIfSystemProperty(
+            named = "skipSparkTest",
+            matches = "true"
+    )
+    public void testAggregateWithTokenizerFormatBytes() {
+        final String id = UUID.randomUUID().toString();
+        streamingTestUtil
+                .performDPLTest(
+                        "index=index_A earliest=2020-01-01T00:00:00z latest=2023-01-01T00:00:00z "
+                                + "| teragrep exec tokenizer format bytes "
+                                + "| teragrep exec hdfs save overwrite=true /tmp/pth_10_hdfs/aggregatorTokenBytes/"
+                                + id,
+                        aggregateFile, ds -> {
+                        }
+                );
+        this.streamingTestUtil.setUp();
+        streamingTestUtil
+                .performDPLTest(
+                        "| teragrep exec hdfs load /tmp/pth_10_hdfs/aggregatorTokenBytes/" + id + " "
+                                + "| teragrep exec bloom estimate "
+                                + "| teragrep exec hdfs save overwrite=true /tmp/pth_10_hdfs/aggregatorEstimate/" + id,
+                        aggregateFile, ds -> {
+                        }
+                );
+        this.streamingTestUtil.setUp();
+        streamingTestUtil
+                .performDPLTest(
+                        "| teragrep exec hdfs load /tmp/pth_10_hdfs/aggregatorTokenBytes/" + id
+                                + "| join type=inner max=0 partition [| teragrep exec hdfs load /tmp/pth_10_hdfs/aggregatorEstimate/"
+                                + id,
+                        aggregateFile, ds -> {
+                            Config config = ConfigFactory.parseProperties(getDefaultProperties());
+                            TeragrepBloomStep step = new TeragrepBloomStep(
```

Comment:
There was no dpl command for aggregate since it is just run before  `| teragrep  exec bloom create `. 
I just wanted to test the aggregate step here. Not sure if there is a better way to do it? I also need to provide a custom config for the step.

Reply:
In that case this is probably the best option available right now. Would remove the get prefix from the getDefaultProperties method.

Reply:
Refactored and removed the helper method it was a bit unnecessary